### ISSUE TITLE
Renishaw reader calculates the scale for the microscope image incorrectly

### ISSUE THREAD

**Issue by Asharits** [hyperspy/rosettasciio#301]
#### Describe the bug

In the Renishaw reader, the scale for the microscope image (WHTL) is being calculated incorrectly for hyperspectral maps. This causes the mapped area (red box) to show up in the wrong area. 

#### To Reproduce

I verified this by comparing images to mapped area in the WiRE versus what Hyperspy is plotting for multiple datasets.

#### Expected behavior

I am not sure where in the reader this is being calculated but it is an easy fix. In the original metadata for the microscope image, FocalPlaneXYOrigins contains the origin for X and Y. FocalPlaneXResolution and FocalPlaneYResolution contains the length of the image. These values just need to be added to the origin values and then divided by the image shape to obtain the correct scale.

I tested this out on some of my datasets and it fixed the scale and the red box was now in the correct place.

#### Python environment:
 - RosettaSciIO version: 0.6
 - Python version: 3.11.8
 - HyperSpy version: 2.1.1

**Comment by ericpre** [comment]
Are you saying that the offset and scale of the microscope image are incorrect but the hyperspectral maps are correct?

The scale and offset of the microscope image are calculated in https://github.com/hyperspy/rosettasciio/blob/e49911047c03465a84facac546053576a23ef915/rsciio/utils/image.py#L42-L70

And what it does seems to be consistent with your description of how it should be calculated.

As I remember, the data that I have doesn't have this issue. Can you share files and screenshots?

**Comment by Asharits** [comment]
Hmm, I wonder why it isn't working for me. Below is a screenshot of the same dataset from WiRE and in Hyperspy. You can see the map are does not match up.

WiRE Screenshot
![Screenshot 2024-08-14 141404](https://github.com/user-attachments/assets/d91677c3-25ac-40d2-ba29-f27ffd22b13d)

Hyperspy Screenshot
![Screenshot 2024-08-14 111531](https://github.com/user-attachments/assets/65c33fed-d8c1-4011-98f8-f8dce83a7803)

**Comment by Asharits** [comment]
To me it looks like an issue with the way scale is calculated using FieldOfViewXY. Maybe there is a difference depending on the version of WiRE? I am on 5.6

**Comment by Asharits** [comment]
@ericpre Looking back at this I misspoke. You only need to change fields_of_views to be equal to (FocalPlaneXResolution, FocalPlaneYResolution). Doing this yields the correct map area for me.

![Figure_2-7_map_test_Signal](https://github.com/user-attachments/assets/af858cb3-170f-4a93-b218-9ecfddc79030)

**Comment by ericpre** [comment]
@Asharits, can you share a file?

**Comment by Asharits** [comment]
Yes, here is a file: [532nm_1800 map.zip](https://github.com/user-attachments/files/17510847/532nm_1800.map.zip)

Here is the screenshot of the file from WiRe: 
![WiRe Screenshot](https://github.com/user-attachments/assets/de47d8f0-5499-4cf2-81fb-5cecd1e8b89d)

Here is what Hyperspy plots before the fix:
![Figure_532nm_1800_map_Signal](https://github.com/user-attachments/assets/6889eef6-c439-49f8-b520-3fb13e1660cc)

Here is it after:
![Figure_532nm_1800_map_Signal2](https://github.com/user-attachments/assets/bf05624f-773b-4876-bce5-338347ddb4c9)

**Comment by ericpre** [comment]
@Asharits, thank you for sharing the file.
Your suggested changes seems to be correct. The data that I used as reference to parse the calibration, have been acquired with WiRE 5.2. The data saved with the two different version of software (WiRE 5,2 and 5.6) doesn't have the same metadata: in case of WiRE 5.2 `FieldOfViewXY` are the same values as (`FocalPlaneXResolution`, `FocalPlaneYResolution`) and therefore doesn't have the issue. For data saved with WiRE 5.6, these values are different!

I will do some check when I have a chance but it all looks consistent. I made a PR with the fix: #327.
